integration-test/build.gradle

```diff
@@ -23,7 +23,8 @@ dependencies {
 			"com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$rootProject.ext.dependencyVersions.jackson",
 			"com.google.code.gson:gson:$rootProject.ext.dependencyVersions.gson",
 			"com.nimbusds:oauth2-oidc-sdk:$rootProject.ext.dependencyVersions.nimbusdsOauth2OidcSdk",
-			"org.slf4j:slf4j-simple:$rootProject.ext.dependencyVersions.slf4j",
+			"org.apache.logging.log4j:log4j-api:$rootProject.ext.dependencyVersions.log4j",
+			"org.apache.logging.log4j:log4j-core:$rootProject.ext.dependencyVersions.log4j",
```

Comment:
I think these come in transitively from powertools-logging?

Reply:
This build.gradle is for the integration tests. I've not added powertools to them as they don't get run as lambdas. Have moved to log4j to be consistent with how we're creating loggers elsewhere, rather than sticking with slf4j.